gh-1583 replace "binary package" with "package manager" (#1644)
tarantool/doc: doc/getting_started/using_package_manager.rst

.. _getting_started-using_package_manager:

--------------------------------------------------------------------------------
Using a package manager
--------------------------------------------------------------------------------

For production purposes, we recommend our

> `official package manager <http://tarantool.org/download.html>`_.

Reviewer: Well, what does this sentence mean? I understand it prior to the changes but now it's just a gibberish.

Author: I agree. My bad. Fixed in 87ab892

> You can choose from tree versions: LTS, stable and beta.

Reviewer 2: ```suggestion
You can choose one of three versions: LTS, stable, or beta.
```

.. code-block:: console

>     $ # if you downloaded a package with apt-get or yum, say this:

Reviewer 3: I guess we should you `installed` or `got` instead of `downloaded`
